families-api/justfile

```diff
@@ -0,0 +1,41 @@
+# dev
+# should be used for first run
+bootstrap:
+    # TODO: find a way to ensure we're using the latest dump
+    @echo "🔄 downloading latest dump from s3"
+    aws s3 cp s3://cpr-production-rds/dumps/2025-04-23-navigator-production.sql ./initial-data/2025-04-23-navigator-production.sql
+
+    @echo "🚀 starting db docker container to load in data"
+    docker compose up -d db
+
+    @echo "⏳ waiting for Postgres to become healthy"
+    @bash just-scripts/is-container-healthy.sh db
+
+    @echo "✅ Bootstrap complete — shutting down"
+    docker compose down
+
+dev:
+    docker compose up
+
+# build
+# TODO: this needs to be tied into a lifecycle somewhere to make sure it is run if something is added
+# sed is used here to remove a whole bunch of gumpf you can not remove with poetry v1 
+requirements:
+    poetry export --without-hashes --without-urls --format requirements.txt | sed 's/ ;.*//'  > requirements.txt
```

Comment:
note: we could add poetry to the `Dockerfile`, but this maintains a slimmer `Dockerfile`, and we can introduce it later if we want.

Reply:
I'm happy with this as is, given that logic of env management is to be able to deal with conflicting dependencies across different projects but by definition the container is its own single python environment. So this keeps principle of simplicity